project/config/uregni/config/config_split.config_split.hosted.yml

```diff
@@ -9,8 +9,7 @@ weight: 0
 folder: ../project/config/uregni/hosted
 module: {  }
 theme: {  }
-blacklist:
-  - search_api.server.solr_default
```

Comment:
Looking at Unity, I think config split needs to have search_api.server.solr_default in all the splits.